baseplate/server/__init__.py

```diff
@@ -433,22 +444,43 @@ def load_and_run_shell() -> None:
 
         ipython_config = Config()
         ipython_config.TerminalInteractiveShell.banner2 = banner
-        start_ipython(argv=[], user_ns=env, config=ipython_config)
+        start_ipython(argv=[f"--logfile=/{console_logpath}"], user_ns=env, config=ipython_config)
         raise SystemExit
     except ImportError:
         import code
 
         newbanner = f"Baseplate Interactive Shell\nPython {sys.version}\n\n"
         banner = newbanner + banner
 
-        # import this just for its side-effects (of enabling nice keyboard
-        # movement while editing text)
         try:
-            import readline
+            _InteractiveConsole_setup(env, console_logpath)
 
-            del readline
         except ImportError:
             pass
 
         shell = code.InteractiveConsole(locals=env)
         shell.interact(banner)
+
+
+def _shell_commands_log_path() -> str:
+    # Define path for console log output
+    pid_1_path = "/proc/1/fd/1"
+    # check if running in a containerized environment
```

Comment:
> containerized environment

Is pretty general, but we're specifically checking if we're in k8s below. Do we care about docker-compose etc? Or just k8s? If the latter, maybe we should call that out specifically?